ostelco-core/Models/SimProfile.swift

```diff
@@ -14,9 +14,19 @@ public enum SimProfileStatus: String, Codable {
     case NOT_READY
 }
 
-public struct SimProfile: Codable {
+public struct SimProfile: Codable, Equatable {
```

Comment:
For this one since `SimProfileStatus` and `String` are both `Equatable`, the compiler auto-generates the implementation.